src/bootstrap/builder.rs

```diff
@@ -686,7 +686,7 @@ impl<'a> Builder<'a> {
                 check::Rls,
                 check::RustAnalyzer,
                 check::Rustfmt,
-                check::Bootstrap
+                check::Bootstrap,
```

Comment:
looks like this is missing the new `BootstrapShim` step?

Reply:
Yeah was debugging it. We have problem at https://github.com/rust-lang/rust/blob/66f8dd12551d11eea0fbcf1bf95b2e18fb07a826/src/bootstrap/builder.rs#L519-L525

because it expects `src/bootstrap/bin/bootstrap-shim` which comes from:

```rs
tool_check_step!(BootstrapShim, "src/bootstrap/bin/bootstrap-shim", SourceType::InTree, false);
```

Reply:
Why not removing `tool_check_step!(BootstrapShim,..` as well ? Seems there is no problem at all with the `x check` command.

Reply:
As long as `x check bootstrap` also checks `bootstrap-shim`, that seems ok to me.